src/index.ts

```diff
@@ -163,11 +196,11 @@ export default class CreditDigital {
       return;
     }
 
-    node.innerHTML = `<div>
-        <a style="color:black;cursor:pointer;" id="credit-digital-checkout">
-          <img src="https://widget.creditdigital.co.uk/paywithcd.png" alt="pay monthly with CreditDigital" />
-        </a>
-      </div>`;
+    node.innerHTML = `
+        <a href="https://www.creditdigital.co.uk" class="creditdigital-checkout">
+          <img src="https://creditdigital-widget.s3.eu-west-2.amazonaws.com/creditdigital-logo-transparent.png" title="credit digital logo" width="50" class="creditdigital-checkout__logo" />
```

Comment:
Do we wanna point this at S3 directly, or pop it behind Cloudfront?

Reply:
Yea I was thinking that... Let me do that instead